scripts/public_api_report.py

```diff
@@ -164,7 +170,11 @@ def _get_names_of_all_usages_in_file(self, file_contents: FileContents) -> Set[s
         import_names = self._get_non_private_gx_import_names(imports=gx_imports)
         logger.debug(f"import_names: {import_names}")
 
-        return function_names | import_names
+        pattern = r"class_name: (\w+)"
```

Comment:
Not sure how often this gets used but you might be able to get a slight performance boost by compiling this regex.

Reply:
Good call! Done :)